examples/ord-min-max.js

```diff
@@ -0,0 +1,42 @@
+const {Max, Min, Ord} = require('../fantasy-monoids');
```

Comment:
Instead of this example I would prefer some unit tests that convey information on usage and behaviour. Unlike unit tests the example also does not contain the output so you have to actually run it and play with it to even know what the result is for sure.

Note that the fantasy checks merely ensure that certain laws are adhered to. That is something quite different.